src/Raven.Server/ServerWide/SecretProtection.cs

```diff
@@ -324,9 +326,116 @@ private static byte[] DecryptProtectedData(byte[] secret, byte[] key)
             return unprotectedData;
         }
 
+        private static void ValidateExpiration(string source, X509Certificate2 loadedCertificate, LicenseType licenseType, SetupProgressAndResult progress = null)
+        {
+            if (loadedCertificate.NotAfter < DateTime.UtcNow)
+            {
+                string msg = $"The provided certificate {loadedCertificate.FriendlyName} from {source} is expired! Thumbprint: {loadedCertificate.Thumbprint}, Expired on: {loadedCertificate.NotAfter}";
```

Comment:
We could add quotes around the name.
```suggestion
                string msg = $"The provided certificate '{loadedCertificate.FriendlyName}' from {source} is expired! Thumbprint: {loadedCertificate.Thumbprint}, Expired on: {loadedCertificate.NotAfter}";
```

Reply:
done